src/Input/index.js

```diff
@@ -72,6 +81,8 @@ Input.propTypes = {
   startIconClass: PropTypes.node,
   /** full `narmi-icon-<shape>` className for icon at end of input */
   endIconClass: PropTypes.node,
+  displayClearInputIcon: PropTypes.bool,
```

Comment:
Optional/minor naming suggestion for this prop. It's an icon that does a thing, so "button" in the name may be more clear

```suggestion
  showClearButton: PropTypes.bool,
```